racket/src/cs/core/memory.ss

```diff
@@ -39,18 +41,31 @@
   
 (define collect-garbage
   (case-lambda
-   [() (collect-garbage 'major)]
+   [()
```

Comment:
Can this stay just `(collect-garbage 'major)`, instead of duplicating the thread-0 condition?

Reply:
Yeah, an oversight on my part.